doc/account.md

```diff
@@ -33,31 +36,55 @@ The `passwordSaltLength` is used to adjust the length of the password salt we ge
         "initiateLifetime": {
             "hours": 1
         },
+        "loginDir": "/login/",
+        "loginIdLength": 24,
+        "loginLifetime": {
+            "minutes": 15
+        },
+        passwordHash: {
+            "algo": "sha512",
+            "hashLength": 48,
+            "iterations": 100000,
+            "salt": ""
+        },
         "passwordSaltLength": 128,
         "registrationDir": "registration/",
         "registrationIdLength": 24
     }
 
+## Signing Up
+
+### Initiating an Account
 
-Initiating an Account
----------------------
 
 The first action for an account is `initiate`. This sets up a few pieces of data needed to properly generate and track an account. We generate a `regId`, `mfaKey`, and `passwordSalt`. An email address must be passed in. These are then passed back to the user to store on their own and will be used to generate their password.
 
 Information will be saved to a file named after the hash of the `regId` in the `registrationDir` [storage] location. This registration file will be deleted upon successful creation of an account file.
 
-Confirming an Account
----------------------
+### Confirming an Account
 
 The next action is to confirm an account. This requires the user to access the URL using the `regId` we gave them. We will then compare it's valid by comparing the hashes then return the `mfaKey` and `passwordSalt` so they can proceed to the next step.
 
-Completing an Account
----------------------
+### Completing an Account
 
 To complete an account, the client will need to pass in a hashed `password`, the `regId` which we gave them previously as well as a `currentMfa` code and `previousMfa` code. This will allow us to find the registration file using the hashed `regId` and looking it up in [storage]. Getting the `mfaKey` off the account we are able to then validate the user using the MFA codes they passed in.
 
 If the account validates, we create the account file from the registration file, add the `password` to it, and generate an `accountId`. The file is then saved under the `accountDir` using the hashed `accountId` as the file name. Only the `accountId` is sent back to the client.
 
+## Logging In
```

Comment:
I do not feel that this dives into the technical details enough.  I don't think I could write a client that understands how to log in with only this documentation.